programs/disks/DisksClient.cpp

```diff
@@ -0,0 +1,290 @@
+#include "DisksClient.h"
+#include <Client/ClientBase.h>
+#include <Client/ReplxxLineReader.h>
+#include <Parsers/parseQuery.h>
+#include <Poco/Util/HelpFormatter.h>
+#include <Common/Config/ConfigProcessor.h>
+#include <Common/EventNotifier.h>
+#include <Common/ZooKeeper/ZooKeeper.h>
+#include <Common/filesystemHelpers.h>
+
+#include <Disks/registerDisks.h>
+
+#include <Formats/registerFormats.h>
+#include <Common/TerminalSize.h>
+
+namespace ErrorCodes
+{
+extern const int BAD_ARGUMENTS;
+extern const int LOGICAL_ERROR;
```

Comment:
My automatic set up clang-format does not add indent here. I think this is a bad practice to leave code which style contradicts to .clang-format

Reply:
It is quite hard to correct style check issues that contradicts to .clang-format (because I have set up tooling for code style of newly added or modified code). You can call the third party to resolve our misunderstanding